Bio/PDB/PDBList.py

```diff
@@ -23,8 +23,11 @@
 #   homepage : http://jaceksmietanski.net
 #   email    : jacek.smietanski@ii.uj.edu.pl
 #
-# This code is released under the conditions of the Biopython license.
-# It may be distributed freely with respect to the original author.
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+# It may be distributed freely with respect to the original authors.
```

Comment:
We've not actually started dual licensing yet (see #898) as I'm still hoping to get a lawyer to look at the proposed wording for the header.